frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx

```diff
@@ -245,10 +245,6 @@ class Visualization extends PureComponent {
   }
 
   visualizationIsClickable = clicked => {
-    const { onChangeCardAndRun } = this.props;
-    if (!onChangeCardAndRun) {
-      return false;
-    }
```

Comment:
Should be safe to remove this; traced this back to https://github.com/metabase/metabase/pull/4496 and I don't see unwanted side effects.